### PR TITLE
Version 0 1 2dev0

### PR DESCRIPTION
**Pull request by nathanfranklin**

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -59,9 +59,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.1'
+version = '0.1.2'
 # The full version, including alpha/beta/rc tags.
-release = '0.1.1.dev0'
+release = '0.1.2dev0'
```
Comment (itziakos) [inline doc/source/conf.py]: this should be 0.1.2.dev0

Comment (nathanfranklin) [inline doc/source/conf.py]: its correct after the second commit, right?

Comment (itziakos) [inline doc/source/conf.py]: The error is that you do not have a `.` before dev

**Comment by itziakos** [comment]
In general this PR should happen after the pre-release PR is merged and the release has been tagged, so that the history of the commits is proper.

**Comment by nathanfranklin** [comment]
@itziakos  Thanks.  I will cancel this PR and do that.

File: doc/source/conf.py (hunk 1 above)
Comment (nathanfranklin) [inline doc/source/conf.py]: thanks

**Pull request closed without merging**